hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/resource/AttributeSearchResource.kt

```diff
@@ -48,6 +48,10 @@ class AttributeSearchResource(private val attributeSearchService: AttributeSearc
       Combining multiple sub-queries gives us the ability to create complex searches using any combination of a prisoner's 
       attributes. For example we can model nested queries such as <strong>"lastName IS Smith AND (prisonId IS MDI OR (prisonId IS OUT AND inOutStatus is OUT))"</strong>.
       </p>
+      <p>Please be aware that when searching lists of complex objects (e.g. aliases, alerts, tattoos) if you want to search for multiple attributes within the same object then you need 
+      to include them in the same query. For example, to search for alias "John Smith" you should search for aliases.firstName IS "John" and aliases.lastName IS "Smith" in a single query.
+      If you search for them in different queries you will receive anyone with firstName John and also anyone with lastName Smith.
+      </p>
```

Comment:
The main reason we have this single query rule is to avoid complexity in the code - otherwise in a complicated scenario with multiple queries and sub-queries and various ANDs / ORs it's really hard to know when to use a nested query for the attributes from nested objects. Just writing tests for it would be horrible. So we've compromised on this rule.